rollingpin/main.py

```diff
@@ -137,8 +137,8 @@ def _select_hosts(config, args):
     hosts = interleaved(all_hosts_unsorted, key=lambda h: h.pool)
 
     try:
-        aliases = resolve_aliases(config["aliases"], hosts)
-        full_hostlist = resolve_hostlist(args.host_refs, hosts, aliases)
+        full_hostlist = resolve_hostlist(
+            args.host_refs, hosts, config["aliases"])
```

Comment:
This whole config thing could probably be moved to the baseplate config system (it predates that and is a sorta proto-version of it) and cleaned up so we don't use magical string indexes.